### ISSUE TITLE
dataTables.bootstrap.js should not get flagged as a duplicate copy of bootstrap.js

### ISSUE THREAD

**Issue by cvrebert** [twbs/bootlint#60]
``` html
<script type="text/javascript" src="/static/js/dataTables.bootstrap.js"></script>
```

currently results in this false positive:

```
bootlint: Only one copy of Bootstrap's JS should be included; currently the webpage includes both bootstrap.js and bootstrap.min.js 
```
